airflow/cli/cli_config.py

```diff
@@ -500,6 +500,18 @@ def string_lower_type(val):
     help="Drop the archive tables after exporting. Use with caution.",
     action="store_true",
 )
+ARG_DB_RETRY = Arg(
+    ("--retry",),
+    default=0,
+    type=positive_int(allow_zero=True),
+    help="Retry database check upon failure",
+)
+ARG_DB_RETRY_DELAY = Arg(
+    ("--retry-delay",),
+    default=1,
```

Comment:
I think we should give the database more time to breathe. :) 

Would it make sense to increase the default value to 30 or something more?

Reply:
The db check retry here is really targetted to see if the database is "up" already - nothing more.

I think 1s is quite good as a default. We do not do a LOT - we just try to connect to the database which takes milliseconds usually.  And if it does not work, then it usually means that the database is not yet ready and we will simply fail on failed "connection" establishing - nothing that makes it "heavy". 

The longer the delay is, the longer we will delay startup if this command is used as a prerequisite to start airflow component (and this is the intention).

Reply:
Sure, that makes sense. :)